src/Forms/PluginsGet.cs

```diff
@@ -377,14 +375,10 @@ private void buttonUpdateAll_Click(object sender, EventArgs e)
                     wc.DownloadDataCompleted += wc_DownloadDataCompleted;
                     wc.DownloadDataAsync(new Uri(_updateAllListUrls[i]));
                 }
-                Cursor = Cursors.Default;

```

Comment:
Shouldn't be here... it will be called just after the line # 378 finishes creating async task.